src/orchestrator-tracing/src/lib.rs

```diff
@@ -179,6 +179,17 @@ pub struct TracingCliArgs {
     /// See: <https://docs.sentry.io/platforms/rust/configuration/options/#environment>
```

Comment:
Can we adjust the docs on `--opentelemetry-resource` above to say "Also used for Sentry tags, regardless of whether `--opentelemetry-endpoint` is specified"?

Reply:
that seems fine as well